kgitb/github.py

```diff
@@ -12,34 +12,34 @@
 
 
 def protect(string):
-    return string.replace("[", r"\[")
+    return string.replace('[', r'\[')
 
 
 def quote(string):
-    return ">" + string.replace("\n", "\n>")
+    return '>' + string.replace('\n', '\n>')
 
 
 def get_commits(url):
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a docstring to this function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)